src/main/java/io/fabric8/maven/docker/BuildMojo.java

```diff
@@ -26,6 +26,9 @@
     @Parameter(property = "docker.skip.build", defaultValue = "false")
     protected boolean skipBuild;
 
+    @Parameter(property = "docker.pull.retries", defaultValue = "0")
+    private int retries;
```

Comment:
Could we please rename this in `pullRetries` for the member configuration ? "retries" alone is a bit unspecific.

Reply:
Same for the other MoJos please and the documentation.